Cargo.toml

```diff
@@ -10,3 +10,11 @@ documentation = "http://alexcrichton.com/rustc-demangle"
 description = """
 Rust compiler symbol demangling.
 """
+
+[workspace]
+members = ["rustc-demangle-capi"]
+
+[profile.release]
+lto = true
+codegen-units = 1
```

Comment:
With `lto = true` this and `incremental = false` shouldn't be necessary to specify as well